sdk/communication/ci.yml

```diff
@@ -55,3 +55,5 @@ extends:
         safeName: azurecommunicationjobrouter
       - name: azure-tools-communication-toll-free-verification
         safeName: azuretoolscommunicationtollfreeverification
+      - name: azure-tools-communication-verified-recipients
+        safeName: azuretoolscommunicationverifiedrecipients
```

Comment:
@classBabacar-msft  is this package intended only for portal consumption?
I ask because for short codes and alpha ids, for example, we are trying to remove the live documentation,  and we may need a couple of flags here so it is not published, please verify that too.

skipPublishDocMs: true
skipPublishDocGithubIo: true